server/lib/util.js

```diff
@@ -264,3 +267,28 @@ export function getCSPNonce(res : ExpressResponse) : string {
 
     return nonce;
 }
+
+/**
+* Handler for server error and communication with the parent window.
+* Return a HTML including the JS SDK allowing the communication with the parent SDK.
+*/
+export function htmlErrorHandler({ res, meta, errorMessage } : ErrorArgument) {
+    const cspNonce = getCSPNonce(res);
+    const pageHtml = `<!DOCTYPE html>
+        <html>
+            <body data-nonce="${ cspNonce }">
+                ${ meta.getSDKLoader({ nonce: cspNonce }) }
+                <script nonce="${ cspNonce }">
+                    const error = new Error("${ errorMessage }");
```

Comment:
Let's use `var` here to be safe. This code does not go through a transpiler so let's make sure it only uses ES3 features.
```suggestion
                    var error = new Error("${ errorMessage }");
```